sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -753,8 +757,12 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
 
                       {listing?.status === ListingStatus.closed && (
                         <LotteryResults
-                          submitCallback={() => {
-                            triggerSubmitWithStatus(false, ListingStatus.closed)
+                          submitCallback={(data) => {
+                            triggerSubmitWithStatus(
+                              false,
+                              ListingStatus.closed,
+                              data as FormListing
```

Comment:
I think it would be better to update `newData ` type to `FormListing  | ListingEvent[]`, instead of use type assertion, because it's a little bit confusing inside the `triggerSubmitWithStatus ` function. What do you think?

Reply:
I'm not sure I follow…all `newData` is doing here is providing a way to override/merge any form data provided by either defaults or React Hook Form. Maybe we're only using it for events data currently, but I don't think it's a given that's all we'll ever do.

Reply:
I see, so maybe we should update `newData` to: `Partial<FormListing>`?

Reply:
Sounds good, I'll try that.